src/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface NotificationsResponse {
+  notifications: Notification[]
+}
+
+interface Notification {
+  notificationCode: number
+  title: string
+  type: string
+  contents: string
+  readStatus: string
+  startDate: Date
```

Comment:
```suggestion
  startDate: string
```
明示的に`Date`に変換しない限り`string`だと思う